string wlan_ip
string motor_direction
int32 pan_degree
int32 tilt_degree
